app.js

```diff
@@ -1,14 +1,34 @@
 const express = require('express')
-const { getCategories } = require('./Controllers/Get-Categories')
+const { getCategories } = require('./Controllers/Get-Categories');
+const { getReviews } = require('./Controllers/Get-reviews');
 
 
 const app = express();
 
 app.use(express.json());
```

Comment:
Just a note - we shouldn't see app.use(express.json()) at this point, no ticket that you're currently working on requires it.